webplugin/css/app/mck-sidebox-1.0.css

```diff
@@ -3051,6 +3051,17 @@ display: none;
 .mck-group-inner .mck-msg-left .mck-msg-box:after {
 	border-right-color: #e6e5ec;
 }
+.message-class-left {
```

Comment:
use some other naming, remove class and also see alternative for left
reason is that in RTL what comes on left will be on right side, and this naming might confuse us in future.